lib/rules/jsx-no-literals.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
 
     function getValidation(node) {
       const standard = !/^[\s]+$/.test(node.value) &&
+          typeof node.value === 'string' &&
```

Comment:
Interesting. Arrays and objects and regexes and functions are all literals too, so it's weird they wouldn't show up. Thanks for explaining.